microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -88,11 +80,11 @@ Your download is a `.zip` archive that contains an `.msixbundle` file and a `.cl
 
 To submit your app to the Microsoft Store:
 
-1.  Go to [Windows Partner Center](https://partner.microsoft.com/dashboard/windows/overview)
+1.  Go to [Microsoft Partner Center](https://partner.microsoft.com/dashboard/windows/overview).
 1.  Select your app.
 1.  Select **Start your Submission**.
```

Comment:
```suggestion
1.  Select **Start your submission**.
```
to match UI, rendered:
https://github.com/zateutsch/edge-developer/blob/2482-updated-pwa-partner-center-guidance/microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md#submit-your-app-package-to-the-store